samples/grids/grid/column-resizing-styling/App.razor

```diff
@@ -0,0 +1,93 @@
+@using IgniteUI.Blazor.Controls
+
+@inject IJSRuntime JS
+
+<style>
+	.grid {
+		--igx-grid-resize-line-color: #f35b04;
+	}
+</style>
+
+<div class="container vertical">
+
+	<div class="container vertical fill">
+		<IgbGrid AutoGenerate="false"
+				 Data="AthletesData"
+				 Id="grid"
+				 Name="grid"
+				 @ref="grid"
+				 Class="grid">
+			<IgbColumn Name="ID"
+					   @ref="iD"
+					   Field="ID"
```

Comment:
First column appears empty:
![image](https://user-images.githubusercontent.com/10397980/235954862-48f60036-1396-4cad-82ca-f832e6ab2e87.png)
Probably the Field name does not match with the data.